api/filter_outputs.go

```diff
@@ -0,0 +1,28 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+func (api *API) createFilterOutput(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	var request createFilterOutputRequest
+	if err := api.ParseRequest(r.Body, &request); err != nil {
+		api.respond.Error(ctx, w, http.StatusBadRequest, errors.Wrap(err, "parse request"))
+		return
+	}
+	if err := api.validate.Struct(&request); err != nil {
+		api.respond.Error(ctx, w, http.StatusBadRequest, errors.Wrap(err, "validate request fields"))
+		return
+	}
+
+	if err := api.store.CreateFilterOutput(ctx, request.Downloads); err != nil {
+		api.respond.Error(ctx, w, http.StatusBadRequest, errors.Wrap(err, "upsert filter output"))
```

Comment:
StatusBadRequest?

Reply:
Sorry, should be StatusInternalServerError